test/tests/lift.jl

```diff
@@ -5,7 +5,7 @@ struct MyType
   x
 end
 
-Omega.lift(:MyType; n=1, mod=Main)
+Omega.lift(:MyType; n=1, mod=Main) # FIXME: Make this work with TestNamespace
```

Comment:
@zenna  I was unable to move this file into its own namespace because of this line. I'd like your help doing so.